multi-agent/adapters/inbound/flask/endpoints/sessions.py

```
@@ -246,12 +248,80 @@ def get_session_status(session_id):
+    """Parse and allowlist the ``filters`` query param.
+
+    Returns the parsed dict, or raises ``ValueError`` with a client-facing
+    message if the value is malformed or contains unsupported keys.
+    """
+    if not raw_filters:
+        return {}
+
+    try:
+        parsed = json.loads(raw_filters)
+    except (json.JSONDecodeError, TypeError) as e:
+        raise ValueError("filters must be a valid JSON object") from e
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)